src/reducers/comments.spec.tsx

```diff
@@ -637,4 +641,287 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('abortLoadVersionComments', () => {
+    it('sets isLoading to false', () => {
+      const versionId = 1;
+      let state;
+      state = reducer(state, actions.beginLoadVersionComments({ versionId }));
```

Comment:
It might be helpful to check `isLoading` between these two calls (although I am sure one test case somewhere makes sure `beginLoadVersionComments()` sets `isLoading` to `true`).

Reply:
That's already covered by this test: https://github.com/mozilla/addons-code-manager/pull/1092/files/945c2d0f5e8d33a145b2a2db7fc0369da42c3780#diff-680427ffbd1269509fdb6a0d3a6346b1R664